components/fatfs/Kconfig

```diff
@@ -105,4 +105,19 @@ config FATFS_MAX_LFN
    help
       Maximum long filename length. Can be reduced to save RAM.
 
+choice FATFS_TINY
+   prompt "Use per-file caches"
```

Comment:
Good catch noticing this option!

Could you please make this a "bool" option CONFIG_FATFS_PER_FILE_CACHES rather than a "prompt"? Can do the necessary inverting of logic (tiny mode == !per file caches) in the header file if necessary.